workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/CloudProvisioning/components/ProvisioningTable.tsx

```diff
@@ -83,8 +87,16 @@ const ProvisioningTable: React.FC<ProvisioningTableProps> = ({
 		navigate(`/order/${order?.id}/cloud-provisioning/install`);
 	};
 
-	const uninstall = async () => {
+	const uninstall = async (detailsData: DetailsData) => {
+		setLoading(true);
+
 		try {
+			await consoleOAuth2.uninstallApp(detailsData.order.id, {
+				orderItemId: detailsData.orderItem.orderItem,
+				projectId:
```

Comment:
Based on this comment

https://github.com/is-solutions-delivery/liferay-portal/pull/5998#discussion_r1786796026

please use the `id:`

instead of using projectId